### DISCUSSION TITLE
MHP30 PID range?

### DISCUSSION THREAD

**Discussion by bradanlane** [Ralim/IronOS#1303] [Q&A]
What is the PID range for the MHP30? 

I have the temp set to 250C. It warms up to 257C (or a bit more), then cools to 240C, then back to 257C, endlessly.

is a 17C swing within expectations?

**Comment by Ralim** [comment]
That is not expected.
You will see that if you put a large cold object on it but with out anything on the hot plate it should hold +- 2C

What power supply are you using?
Do you have power limits set in the menu?

**Comment by bradanlane** [comment]
Using the manufacturer's power supply  65W? I think.

no power limit settings. Still all default values from initial firmware install.

I set to 240C and the swing is the same; about -10C to +7C. Same results with nothing on the hotplate as with a dummy PCB on top.

happy to run others tests; change settings; etc.
